clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go

```diff
@@ -59,6 +59,16 @@ const (
 	// countQuery %v should be replaced with (1) filters and (2) query window size.
 	countQuery = "sum(increase(apiserver_request_duration_seconds_count{%v}[%v])) by (resource, subresource, scope, verb)"
 
+	countSlowQuery = "sum(rate(apiserver_request_duration_seconds_bucket{%v}[%v])) by (resource, subresource, scope, verb)"
+
+	// Bucket definitions: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go#L84
+	// exclude all buckets of 1s and shorter
+	filterGetAndMutating = filters + `, verb!="LIST", le!~"0.\\d+|1"`
+	// include all buckets above 5s
+	filterNamespaceList = `resource!="events", scope!="cluster", verb="LIST", le=~"[6789]|\\d0|\\+Inf"`
```

Comment:
It's not strictly related to this PR, but I suggest removing "resource!=events" finally.
We added it 5 years ago, because we have 1s for every opertation back then and there were too many events.
But currently, this is no longer the case, so let's remove events.
@mm4tt - FYI

Reply:
Might be a separate PR to remove it from the existing place (and then got rid of it here in this PR).

Reply:
I am going to remove it now then.

Reply:
https://github.com/kubernetes/perf-tests/pull/1237